.github/workflows/pull.yml

```diff
@@ -9,9 +9,11 @@ on:
       - release/*
       - landchecks/*
   workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
nit: shall we move this to something slightly later, like 3:30am, so that we stagger our night time periodic jobs a bit?

Reply:
moved it to about 1:30am pst